test/test_util_error.cpp

```diff
@@ -64,7 +64,12 @@ TEST(BasicSystemErrors_Category)
 
 TEST(BasicSystemErrors_Messages)
 {
+#if defined(__linux__) && !defined(__GLIBC__)
```

Comment:
it could be nice to make a REALM_MUSL macro in features.h for readability

Reply:
I wouldn't do that outside of tests as there isn't a sure-fire way to detect musl based on macros alone - in this particular case we know we only build the tests against glibc and musl on Linux, but if someone were to build Core itself with something like uclibc, for example, it would trigger the above code path if it were in Core itself and not just in the tests we build and run ourselves.